sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryIOWriteTest.java

```diff
@@ -302,15 +303,15 @@ public void testWriteDynamicDestinations() throws Exception {
   }
 
   @Test
-  public void testWriteDynamicDestinationsBatchWithSchemas() throws Exception {
-    writeDynamicDestinations(true, false);
+  public void testWriteDynamicDestinationsWithAutoSharding() throws Exception {
+    assumeTrue(useStreaming);
+    assumeTrue(!useStorageApiApproximate); // STORAGE_API_AT_LEAST_ONCE ignores auto-sharding
+    writeDynamicDestinations(false, false);
   }
 
   @Test
-  public void testWriteDynamicDestinationsStreamingWithAutoSharding() throws Exception {
-    assumeTrue(useStreaming);
-    assumeTrue(!useStorageApiApproximate); // STORAGE_API_AT_LEAST_ONCE ignores auto-sharding
-    writeDynamicDestinations(true, true);
+  public void testWriteDynamicDestinationsWithBeamSchemas() throws Exception {
```

Comment:
Why did we change the name of this?

Reply:
To be more accurate:
1. This tests both batch and streaming modes
2. `withSchemas` actually means using with Beam Schema